src/plugins/multimedia/multimedia.js

```diff
@@ -1064,14 +1086,31 @@ $document.on( multimediaEvents, selector, function( event, simulated ) {
 			if ( isPlay ) {
 				$this.addClass( "playing" );
 				$this.find( ".progress" ).addClass( "active" );
+
```

Comment:
The attribute "aria-pressed" should be set to true on page load. Not only when the video starts. The `<figure>` element should also have the class "cc_on" so the styles of the button are applied.